.editorconfig

```diff
@@ -1,22 +1,20 @@
 # EditorConfig helps developers define and maintain consistent
 # coding styles between different editors and IDEs
 # editorconfig.org
-
 root = true
 
-
 [*]
-
-# Change these settings to your own preference
```

Comment:
Why removing this comments?

Reply:
Because you're not supposed to change theses settings according to your preferences 😃